.github/workflows/sel4test-hw.yml

```diff
@@ -33,11 +33,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        march: [armv7a, armv8a, nehalem]
+        march: [armv7a, armv8a, nehalem, rv32imac, rv64imac]
```

Comment:
Would also leave out `rv32imac` here, since we currently have no hardware for it.